plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -534,6 +534,20 @@ class FossId internal constructor(
             .checkResponse("download data from Git", false)
 
         if (existingScan == null) {
+            val excludesRules = context.excludes?.let {
```

Comment:
Commit message:
- "This commit change*s*"
- "in this case" should probably say "in the other case"